The default value of the crs field is required #152 (#154)
opengeospatial/geoparquet: format-specs/geoparquet.md

Apart from the difference of encodings, the semantics are intended to match
WKT2:2019, and a CRS in one encoding can generally be represented in the other.

> all coordinates in the geometries must use longitude, latitude based on the WGS84 datum

Author: ```suggestion
If CRS is not provided, all coordinates in the geometries must use longitude, latitude based on the WGS84 datum
```

> If CRS is not provided, all coordinates in the geometries must use longitude, latitude based on the WGS84 datum

Reviewer: ```suggestion
If CRS is not provided, all coordinates in the geometries must use longitude, latitude based on the WGS84 datum,
```

> If the `crs` field is not provided, the default CRS is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84),

Reviewer 2: ```suggestion
If the `crs` field is not provided, all coordinates in the geometries must use longitude, latitude based on the WGS84 datum,
```

Author: all coordinates in the geometries must use longitude, latitude based on the WGS84 datum

Author: ```suggestion
all coordinates in the geometries must use longitude, latitude based on the WGS84 datum
```

> which means that all coordinates in the geometries must use longitude, latitude based on the WGS84 datum.

Reviewer 2: ```suggestion
which means CRS-aware implementations can assume a default value of [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84)
```

I think switching the order as suggested in these two lines can get across that if you are implementing and don't know about CRS's then you can just use long/lat. I do think it'd be good to not make it seem like to implement geoparquet you need a full projection engine - you could just read and write only long/lat. 

If we were to do a full OGC-style spec we'd likely have the core conformance just be long/lat, and another conformance class add in more projections (crs field and ability to understand it)

Author: We tried to avoid the "can assume" wording as this is weird, so another proposal:

```suggestion
which means the default value is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84) for CRS-aware implementations
```

> [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84) is equivalent to the well-known [EPSG:4326](https://epsg.org/crs_4326/WGS-84.html) but changes the axis from latitude-longitude to longitude-latitude.

Reviewer 2: No issue. That twisted language came from some idea that there is data that people put in lat/long that is not exactly crs84, due to some sub-parameters of the WKT, so this was to somehow communicate that it doesn't need to be exact, or something... I forget the exact thing that lead to this. But since I can barely remember what we were trying to achieve I'm in support of clearer language.

> [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84) is equivalent to the well-known [EPSG:4326](https://epsg.org/crs_4326/WGS-84.html) but changes the axis from latitude-longitude to longitude-latitude.

Author: Could you have a look again? I tried to squeeze out the "can assume" while trying to keep your intention.
